orttraining/orttraining/test/python/orttraining_ortmodule_tests.py

```diff
@@ -37,6 +37,14 @@ def run_ortmodule_poc_net(cwd, log):
     run_subprocess(command, cwd=cwd, log=log).check_returncode()
 
 
+def run_ortmodule_bert_classifier(cwd, log):
```

Comment:
there are lots of bert classifier out there. lets try to be more specific to help differentiating them in the future

`run_ort_module_hf_bert_for_sequence_classification_from_pretrained`